src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/Contracts/NodeInfo.cs

```diff
@@ -25,6 +25,11 @@ public class NodeInfo
         /// </summary>
         public string NodePath { get; set; }
 
+        /// <summary>
+        /// The path of the parent node.

```

Comment:
I would suggest putting a remark here though that this intended to allow nodes to look up their parents - just to make it clear that we aren't trying to store all the parent node info here.